src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.tsx

```diff
@@ -1,108 +1,87 @@
 import {trackAnalyticsEvent} from 'app/utils/analytics';
-import SentryTypes from 'app/sentryTypes';
 import withOrganization from 'app/utils/withOrganization';
 import Confirmation from 'app/components/onboardingWizard/confirmation';
 import InlineSvg from 'app/components/inlineSvg';
 import Button from 'app/components/button';
 import space from 'app/styles/space';
+import ExternalLink from 'app/components/links/externalLink';
+import {OnboardingTask, Organization} from 'app/types';
+import {navigateTo} from 'app/actionCreators/navigation';
 
-class TodoItem extends React.Component {
-  static propTypes = {
-    task: PropTypes.object.isRequired,
-    onSkip: PropTypes.func.isRequired,
-    organization: SentryTypes.Organization,
-  };
+type Props = ReactRouter.WithRouterProps & {
+  task: OnboardingTask;
+  onSkip: (taskKey: number) => void;
+  organization: Organization;
+};
 
-  state = {
-    showConfirmation: false,
-    isExpanded: false,
-  };
+type State = {
+  showConfirmation: boolean;
+};
 
-  toggleDescription = () => {
-    this.setState({isExpanded: !this.state.isExpanded});
+class TodoItem extends React.Component<Props, State> {
+  state: State = {
+    showConfirmation: false,
   };
 
-  toggleConfirmation = () => {
+  toggleConfirmation = (e: React.MouseEvent) => {
     this.setState({showConfirmation: !this.state.showConfirmation});
+    e.preventDefault();
   };
 
-  formatDescription() {
-    const {task} = this.props;
-    const {isExpanded, showConfirmation} = this.state;
-
-    return (
-      <React.Fragment>
-        {task.description}
-        {(isExpanded || showConfirmation) && '. ' + task.detailedDescription}
-      </React.Fragment>
-    );
-  }
-
-  learnMoreUrlCreator() {
-    const org = this.props.organization;
+  get formattedDescription() {
     const {task} = this.props;
-    let learnMoreUrl;
-    if (task.featureLocation === 'project') {
-      learnMoreUrl = `/organizations/${org.slug}/projects/choose/?onboarding=1&task=${task.task}`;
-    } else if (task.featureLocation === 'organization') {
-      learnMoreUrl = `/organizations/${org.slug}/${task.location}`;
-    } else if (task.featureLocation === 'absolute') {
-      learnMoreUrl = task.location;
-    } else if (task.featureLocation === 'modal') {
-      learnMoreUrl = undefined;
-    } else {
-      Sentry.withScope(scope => {
-        scope.setExtra('props', this.props);
-        scope.setExtra('state', this.state);
-        Sentry.captureMessage('No learnMoreUrl created for this featureLocation');
-      });
-    }
-    return learnMoreUrl;
+    return `${task.description}. ${task.detailedDescription}`;
   }
 
-  recordAnalytics(action) {
+  recordAnalytics(action: string) {
     const {organization, task} = this.props;
     trackAnalyticsEvent({
       eventKey: 'onboarding.wizard_clicked',
       eventName: 'Onboarding Wizard Clicked',
       organization_id: organization.id,
-      todo_id: parseInt(task.task, 10),
+      todo_id: task.task,
```

Comment:
Should this be int? https://github.com/getsentry/reload/blob/master/reload_app/events.py#L562

Reply:
It already is an int

Reply:
typescript baby